### ISSUE TITLE
UTF-8 encoding with save_graph_shapefile function error

### ISSUE THREAD

**Issue by raphadasilva** [gboeing/osmnx#21]
Hello Geoff !

I was retesting an interesting case with the new update of save_graph_shapefile function (which includes now an UTF-8 encoding). I'm sure that I was using 0.1 version of OSMnx !

Here's the beginning of my code : 
```

%matplotlib inline
import osmnx as ox, matplotlib.pyplot as plt
from shapely.geometry import Polygon, MultiPolygon, shape
ox.config(log_console=True, use_cache=True)

liege_streets = ox.graph_from_place('Liège, Belgique', network_type='walk', retain_all=True)
liege_streets = ox.project_graph(liege_streets)

fig, ax = ox.plot_graph(liege_streets, fig_height=10, show=False, node_size=0, close=False, edge_color='#777777')

plt.show
```
The street network of Liège is displaying well, everything is cool so far. Then, I want to save these network on the .shp format, with the command : 
`
ox.save_graph_shapefile(rues_liege, filename='rues_liege')`

But then, got this error : 

```
ERROR:Fiona:Failed to encode property 'name' value 'Place Crève Cœur'
---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
<ipython-input-19-d96d6a09437d> in <module>()
----> 1 ox.save_graph_shapefile(rues_liege, filename='rues_liege')

/usr/local/lib/python3.4/dist-packages/osmnx/osmnx.py in save_graph_shapefile(G, filename, folder)
   2070     # save the nodes and edges as separate ESRI shapefiles
   2071     gdf_nodes.to_file('{}/nodes'.format(folder))
-> 2072     gdf_edges.to_file('{}/edges'.format(folder))
   2073     log('Saved graph "{}" to disk as shapefiles at "{}" in {:,.2f} seconds'.format(G_save.name, folder, time.time()-start_time))
   2074 

/usr/local/lib/python3.4/dist-packages/geopandas/geodataframe.py in to_file(self, filename, driver, schema, **kwargs)
    341         """
    342         from geopandas.io.file import to_file
--> 343         to_file(self, filename, driver, schema, **kwargs)
    344 
    345     def to_crs(self, crs=None, epsg=None, inplace=False):

/usr/local/lib/python3.4/dist-packages/geopandas/io/file.py in to_file(df, filename, driver, schema, **kwargs)
     61                     schema=schema, **kwargs) as c:
     62         for feature in df.iterfeatures():
---> 63             c.write(feature)
     64 
     65 

/usr/local/lib/python3.4/dist-packages/fiona/collection.py in write(self, record)
    333     def write(self, record):
    334         """Stages a record for writing to disk."""
--> 335         self.writerecords([record])
    336 
    337     def validate_record(self, record):

/usr/local/lib/python3.4/dist-packages/fiona/collection.py in writerecords(self, records)
    327         if self.mode not in ('a', 'w'):
    328             raise IOError("collection not open for writing")
--> 329         self.session.writerecs(records, self)
    330         self._len = self.session.get_length()
    331         self._bounds = self.session.get_extent()

/usr/local/lib/python3.4/dist-packages/fiona/ogrext.cpython-34m.so in fiona.ogrext.WritingSession.writerecs (fiona/ogrext1.c:17864)()

/usr/local/lib/python3.4/dist-packages/fiona/ogrext.cpython-34m.so in fiona.ogrext.OGRFeatureBuilder.build (fiona/ogrext1.c:7091)()

/usr/local/lib/python3.4/dist-packages/fiona/ogrext.cpython-34m.so in fiona.ogrext.OGRFeatureBuilder.build (fiona/ogrext1.c:6935)()

UnicodeEncodeError: 'latin-1' codec can't encode character '\u0153' in position 13: ordinal not in range(256)
```

PS : the first time I run the command, I haven't got these error but another one more soft. But then I refresh it on my notebook, got the previous one !

**Comment by gboeing** [comment]
I am not getting this error when I run your code. But, either way, this should be fixed by #20 which lets you specify an encoding (such as utf8) as an argument in the `save_graph_shapefile` function. This is in the latest code on GitHub, and will be released soon.

**Comment by gboeing** [comment]
Resolved by [d07ab54](https://github.com/gboeing/osmnx/commit/d07ab54aa248059c93a9d0559083b4d94597c1b6#diff-922d1a4a0e7d12ee2c3469b315ad7990)

v0.2.1 is now available via pip

**Comment by raphadasilva** [comment]
Sorry I thought that the previous changes were already effective. Just tested the new version, and everything worked super fine !

Thank you again :-)

And, by the way : happy new year !!
